itou/www/itou_staff_views/export_utils.py

```diff
@@ -1,8 +1,13 @@
 from django.core.exceptions import ObjectDoesNotExist
+from django.utils import timezone
 
 from itou.companies.enums import ContractType
 
 
+def get_export_ts():
+    return f"{timezone.localdate().strftime('%Y-%m-%d')}_{timezone.localtime().strftime('%H-%M-%S')}"
```

Comment:
```suggestion
    return timezone.localtime().strftime('%Y-%m-%d_%H-%M-%S')
```
qui évite en plus le bug improbable de l'export à minuit avec la date d'hier :sweat_smile: